Snowdog_Components/components/Templates/product-view/_product-view.scss

```diff
@@ -153,7 +153,7 @@
         display: $product-view__price-display;
 
         @include mq($product-view__price-display-bq) {
```

Comment:
is not a typo? shouldn't be `$product-view__price-display-mq` instead?

Reply:
yeah, that would make sense. I was confused what bq is supposed to mean xd

Reply:
XD